examples/cloudsuite/kit/cloudsuite.py

```diff
@@ -0,0 +1,475 @@
+        )
+
+        self.server_platform.comm.shell(
+            command="docker run -dt --net=host --name=database_server db_built"
+        )
+
+        command = "docker logs database_server | tac | awk '/exit/ {exit} 1' | tac"
+
+        while not mariadb_initialized:
+            time.sleep(1)
+
+            ret = self.server_platform.comm.pipe_shell(
+                command=command,
+                print_command=False,
+            )
+
+            if "Starting MariaDB database server mariadbd" in ret:
+                mariadb_initialized = True
+
+    def single_run(  # pylint: disable=arguments-differ
+        self,
+        benchmark_duration_seconds: int,
+        nb_threads: int = 2,
+        **kwargs,
+    ) -> str:
+
+        ip_web_server = self.web_server_platform.comm.get_ipaddress
+
+        self._clean_run()
+        self._build_run(nb_threads)
+
+        ping_values = self.platform.comm.shell(
+            command=f"ping -c 5 {ip_web_server}",
+            print_input=False,
+            print_output=False,
+        )
+
+        environment = self._preload_env(
+            **kwargs,
+        )
+
+        has_container = self.platform.comm.pipe_shell(
+            command="docker container ls -a | grep faban_client | wc -l", ignore_ret_codes=[1]
+        )
+
+        if "1" in has_container:
+            run_command = [
+                "docker",
+                "start",
+                "-i",
+                "faban_client",
+            ]
+        else:
+            run_command = [
+                "docker",
+                "run",
+                "--net=host",
+                "--name=faban_client",
+                "-v",
+                "/home/drc/rchehab/faban_output/:/web20_benchmark/output",
+                "--env-file",
+                "_tmp_benchkit_seeds.txt",
+                "faban_built",
+                f"{ip_web_server}",
+                f"{nb_threads}",
+                f"--ramp-up={benchmark_duration_seconds + 20}",
+                f"--steady={benchmark_duration_seconds}",
+                "--oper=run",
+            ]
+
+        wrapped_run_command, wrapped_environment = self._wrap_command(
+            run_command=run_command,
+            environment=environment,
+            **kwargs,
+        )
+
+        output = self.run_bench_command(
+            run_command=run_command,
+            wrapped_run_command=wrapped_run_command,
+            current_dir="~/",
+            environment=environment,
+            wrapped_environment=wrapped_environment,
+            print_output=False,
+        )
+
+        return f"{ping_values}\n{output}"
+
+    def get_details(  # pylint: disable=arguments-differ
+        self,
+        record_data_dir: PathType,
+    ) -> None:
+        filename = "/home/drc/rchehab/faban_output/TH_*-TM_1000-TY_THINKTIME-DS_fixed/*/detail.xan"
+        filename_dir = "/home/drc/rchehab/faban_output/TH_*-TM_1000-TY_THINKTIME-DS_fixed"
```

Comment:
hardcoded paths here. That won't generalize on other machines.

Reply:
I've also used the `tmp/benchkit_cloudsuite/` path here and in a few other places with hardcoded paths